qfieldsync/gui/package_dialog.py

```diff
@@ -24,7 +24,20 @@
 
 from libqfieldsync.layer import LayerSource
 from libqfieldsync.offline_converter import ExportType, OfflineConverter
-from libqfieldsync.offliners import QgisCoreOffliner
+
+try:
```

Comment:
```suggestion
# TODO this try/catch was added due to module structure changes in QFS 4.8.0. Remove this as enough time has passed since March 2024.
try:
```